src/Traits/SansDaemonWorkerTrait.php

```diff
@@ -71,13 +71,15 @@ protected function shouldRunNextJob()
      */
     protected function isOverMaxExecutionTime($options)
     {
```

Comment:
You should delete the `$options` here as it is not used.

Then you don't need calling `gatherWorkerOptions` at all.

Reply:
Thanks !